lib/parallel_cucumber/version.rb

```diff
@@ -1,3 +1,3 @@
 module ParallelCucumber
-  VERSION = '0.2.18'.freeze
+  VERSION = '0.2.19.pre2'.freeze
```

Comment:
why ```pre2``` ? maybe just ```0.2.19``` ?

Reply:
I would recommend not to bump parallel_cucumber version in branches and to do it only in master

Reply:
reverted